// [[Reason: undefined]] ////

//// [tests/cases/conformance/types/typeRelationships/comparable/equalityWithEnumTypes.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -12,15 +12,17 @@
 declare function f1(v: E1): void;
 declare function f2(v: E2): void;
 
 /// [Errors] ////
 
+equalityWithEnumTypes.ts(9,5): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
+equalityWithEnumTypes.ts(10,5): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 equalityWithEnumTypes.ts(14,9): error TS2367: This comparison appears to be unintentional because the types 'E1' and '0' have no overlap.
 equalityWithEnumTypes.ts(23,9): error TS2367: This comparison appears to be unintentional because the types 'E1' and '3' have no overlap.
 equalityWithEnumTypes.ts(29,9): error TS2367: This comparison appears to be unintentional because the types 'E2' and '0' have no overlap.
 equalityWithEnumTypes.ts(38,9): error TS2367: This comparison appears to be unintentional because the types 'E2' and '3' have no overlap.
 
 
-==== equalityWithEnumTypes.ts (4 errors) ====
+==== equalityWithEnumTypes.ts (6 errors) ====
     // Literal enum type
     enum E1 {
         a = 1,
         b = 2,
@@ -28,9 +30,13 @@
     
     // Numeric enum type
     enum E2 {
         a = 1 << 0,
+        ~
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
         b = 1 << 1
+        ~
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
     }
     
     function f1(v: E1): void {
         if (v !== 0) {  // Error
